src/modules/commander/Commander.hpp

```diff
@@ -110,7 +110,10 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 		(ParamInt<px4::params::COM_POS_FS_GAIN>) _failsafe_pos_gain,
 
 		(ParamInt<px4::params::COM_LOW_BAT_ACT>) _low_bat_action,
-		(ParamFloat<px4::params::COM_DISARM_LAND>) _disarm_when_landed_timeout
+		(ParamFloat<px4::params::COM_DISARM_LAND>) _disarm_when_landed_timeout,
+
+		(ParamInt<px4::params::MC_AIRMODE>) _airmode,
```

Comment:
Could you use the old param API or do something else here? I'd like to preserve the ability to do FW or MC only builds. I'll add test configs to the CI system.

Reply:
Done